recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
-{% set version = "1.2.2" %}
+{% set name = "tzlocal" %}
+{% set version = "1.3" %}
 {% set hash_type = "sha256" %}
-{% set hash_val = "8e7fb9f9301039c36fb0ff0a004ec36705ac46cee99a7373694a838adc8d30b0" %}
+{% set hash_val = "d160c2ce4f8b1831dabfe766bd844cf9012f766539cf84139c2faac5201882ce" %}
 
 package:
-  name: tzlocal
+  name: {{ name }}-{{ version }}.tar.gz
   version: {{ version }}
 
 source:
   fn: tzlocal-{{ version }}.tar.gz
-  url: https://github.com/regebro/tzlocal/archive/{{ version }}.tar.gz
```

Comment:
Thanks. I find copying-n-pasting the URL much easier too without all that `jinja`. At the end of the day we are checking checksums, license files, creating patches, and all that we need to download the tarball locally. While that `jinja` only helps to copy-and-paste to crate another recipe. (I do like the `jinja` for version though, that is the only one that helps to maintain the feedstock.)